modules/op-release-notes-1-9.adoc

```diff
@@ -0,0 +1,131 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+:_content-type: REFERENCE
+[id="op-release-notes-1-9_{context}"]
+= Release notes for {pipelines-title} General Availability 1.9
+
+With this update, {pipelines-title} General Availability (GA) 1.9 is available on {product-title} 4.10, and is planned to be available on {product-title} 4.11 and later.
+
+[id="breaking-changes-1-9_{context}"]
+== Breaking changes
+
+// .Pipelines
+
+* With this update, the `Conditions` custom resource definition (CRD) type has been removed. As an alternative, use the WhenExpressions CRD instead.
+
+* With this update, support for `tekton.dev/v1alpha1` API pipeline resources, such as, Pipeline, PipelineRun, Task, Clustertask, and TaskRun has been removed.
+
+* With this update, the `tkn-pac setup` command has been removed. You can now re-use the functionality provided by the `tkn-pac setup` command by using the `tkn-pac webhook add` and `tkn-pac webhook update-token` commands. You can use the `tkn-pac webhook add` command to re-add a webhook to an existing Git repository. And, you can use the `tkn-pac webhook update-token` command to update the personal provider access token for an existing Secret object in the Git repository.
+
+[id="deprecated-features-1-9_{context}"]
+== Deprecated and removed features
+
+* In the Red Hat OpenShift Pipelines 1.9 release, `ClusterTasks` are deprecated and planned to be removed in a future release. As an alternative, you can use `Cluster Resolver`.
+
+* In the Red Hat OpenShift Pipelines 1.9 release, the `Trigger` object and the `namespaceSelector` field in the `EventListener` specification are deprecated and planned to be removed in a future release.
```

Comment:
`namespaceSelector` field in the `EventListener` specification are deprecated and planned to be removed in a future release.

From where did we get this?

Reply:
Hey @khrm, you had provided the following text in the Deprecated changes of the SME doc so I added this in the RN:

Deprecation Notices
Deprecation Warning: Having both Triggers as well as Namespace-Selector in EventListener Spec is deprecated.

Kindly suggest if you still want this text in the RN.

Reply:
What I meant there is that you can't have Triggers as well as Namespace-Selector field at the same time.

Reply:
@khrm - Could you please provide me some more context with regards to this statement? The statement suggests that both these fields have been deprecated and I have stated the same. The new statement you provided is a little unclear to me.

Reply:
What I meant is that you can't specify both fields in an EventListener at the same time. In future releases, it will fail the validation.
```
spec:
    namespaceSelector:...
    triggers:...
```
Spec above is deprecated.

Spec below are fine:
```
spec:
    namespaceSelector:
```

```
spec:
    triggers:
```

Reply:
@khrm is trying to say that in a same eventlistener spec you cannot have both `namespaceSelector` and ` triggers` field.  The fields are not being deprecated but the usage of both the fields in a single spec file is deprecated, they can used in two different spec seperately.

Reply:
Thanks for the clarification @rupalibehera. I have made the change in the content. Kindly have a look and confirm if it looks good.

Reply:
```
In the Red Hat OpenShift Pipelines 1.9.0 release, the usage of the `triggers` and  `namespaceSelector` field in a single `EventListener` specification is deprecated and planned to be removed in a future release. You can use these fields in different `EventListener` specifications successfully
```

Reply:
What about ``the simulataneous usage of  the `triggers` and  `namespaceSelector` field in a single `EventListener` specification is deprecated and planned to be removed in a future release. You can use these fields in different `EventListener` specifications ``